src/wallet/wallet.cpp

```diff
@@ -3740,6 +3740,12 @@ bool CWallet::InitLoadWallet()
 
     std::string walletFile = GetArg("-wallet", DEFAULT_WALLET_DAT);
```

Comment:
Should this not be "count" i.e for multiple wallet files to be created ? Adding perhaps a for loop to iterate over required num of wallets?